File: src/androidTest/java/pt/joaomneto/titancompanion/TestSTRIDER.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.STAR_STRIDER;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestSTRIDER extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return STAR_STRIDER;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestSTRIDER.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.STAR_STRIDER

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestSTRIDER : TestST() {

    override val gamebook = STAR_STRIDER
}
